docs/2.guide/1.concepts/3.rendering.md

```diff
@@ -69,6 +69,32 @@ If you do use `ssr: false`, you should also place an HTML file in `~/app/spa-loa
 :read-more{title="SPA Loading Template" to="/docs/api/configuration/nuxt-config#spaloadingtemplate"}
 ::
 
+### Deploying a Static Client-Rendered App
+
+If you deploy your app to [static hosting](getting-started/deployment#static-hosting) with the `nuxi generate` or `nuxi build --prerender` commands, then by default, Nuxt will render every page as a separate static HTML file.
```

Comment:
Could possible also add a video alert on top to https://www.youtube.com/watch?v=7Lr0QTP1Ro8 🤔

Reply:
Good idea - happy to make the code suggestion?